target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -39,6 +39,10 @@ ramips_setup_interfaces()
 	ubnt,unifi-nanohd)
 		ucidef_set_interface_lan "lan"
 		;;
+	ubiquiti,edgerouterx|\
+	ubiquiti,edgerouterx-sfp)
+		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth4"
```

Comment:
Since this seemed to have been lost, `Acked-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>` for the edgerouter x patch.

Reply:
Done